content/blog/2025-11-03-urssi-fall-fellowship-ai-in-sci-soft.md

```diff
@@ -0,0 +1,86 @@
+
+2. **Detecting AI-assisted vs. human-authored code**
+
+We’re evaluating multiple classification models that may identify LLM-assisted code through a mix of features and metrics including comments, variable naming, and modularity[^2]. We plan on applying these models across a large-sample of our constructed dataset in order to estimate the prevalence of AI code assistance in scientific research.
+
+[^2]: For example, [Shi et al., 2025](https://doi.org/10.1109/ICSE55347.2025.00005) provide `DetectCodeGPT`. [Xu and Sheng, 2024](https://doi.org/10.1609/aaai.v38i21.30361) provide `AIG Code Detector`. And, [Oedingen et al., 2025]( https://doi.org/10.48550/arXiv.2405.15512) provide a few options across multiple models.
+
+3. **Understand broader impacts on scientific practice**
+
+We plan on using quantitative metrics for code organization (e.g., lines of code, lines of comments), development patterns (e.g., commit burstiness), and collaboration (e.g., contributor specialization) to understand how scientific software development practices as a whole have changed over time, and especially between samples of AI assisted versus human-written code. Finally, we aim to identify the potential benefits (e.g., more documentation and basic reproducibility) and risks (e.g., logical errors and increased code churn) introduced by generative AI in science.
+
+## Why It Matters
+
+The scientific software community has potentially entered an era where code can be rapidly prototyped, deployed, and integrated into research processes. This brings exciting opportunities for productivity and creativity, but also raises challenges for transparency, trust, and correctness.
+
+Our findings will help:
+
+1. Characterize the change in scientific software practices since the advent of LLMs.
+
+2. Inform, engage, and empower the scientific software community—maintainers, developers, and users alike—to engage critically and collaboratively with the growing role of AI-generated code.
+
+3. Encourage reproducible science by improving awareness of how AI contributions shape research codebases.
+
+All datasets, software, and models we create will be open-source, enabling others to build on this work.
+
+## How We’re Engaging the Community
+
+A central part of the project is collaboration with the research software community:
+
+1. **Open Resources:** We’ll publish our dataset, selected detection models, and analysis scripts under a permissive license, along with thorough documentation.
+
+2. **Collaborative Feedback:** We plan on inviting scientific software maintainers and computational researchers to share feedback on integration strategies, best practices, or general collaboration.
+
+3. **Workshops and Presentations:** We plan to share progress and results, focusing on practical takeaways for the community.
+
+If you’re interested in contributing, providing feedback, or testing our tools, please reach out! 
+
```

Comment:
This seems a bit redundant based on the previous section (which says many of the same things)